addon/components/uni-autocomplete.js

```diff
@@ -38,8 +38,17 @@ export default Component.extend({
         return [];
       }
 
-      let options = this.get('options').map((option) => {
-        let matchedValues = this.filterFunction(this.get('searchTextValues'), option);
+      const options = this.options.map((option) => {
+        const matchedValues = this.filterFunction(this.searchTextValues, option);
```

Comment:
Can you use the `this.X` even the self-service is on ember 2.X?

Reply:
even with `this.get()` as this cli-uniq is on 3.5 this the update on self-service will still not work